tests/dirent/test_readdir.c

```diff
@@ -93,9 +93,12 @@ void test() {
   for (i = 0; i < 3; i++) {
     errno = 0;
     ent = readdir(dir);
-    //printf("ent, errno: %p, %d\n", ent, errno);
-    assert(ent);
-    //printf("%d file: %s (%d : %d)\n", i, ent->d_name, ent->d_reclen, sizeof(*ent));
+    if (ent) {
+      fprintf(stderr, "%d file: %s (%d : %lu)\n", i, ent->d_name, ent->d_reclen, sizeof(*ent));
```

Comment:
does the test actually care what this output is?

Reply:
No, all the output that gets checked come after this part of the test.